airflow/www/views.py

```diff
@@ -2288,6 +2291,36 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, LoggingMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
```

Comment:
Why? The template displays NotAvailable if is it None. Are you suggesting displaying an empty string?